chpl/chpl-service/src/main/java/gov/healthit/chpl/certificationId/Validator2015.java

```diff
@@ -111,15 +117,26 @@ protected boolean isCPOEValid() {
     // **********************************************************************
     protected boolean isDPValid() {
         this.counts.put("criteriaDpRequired", 1);
+        
+        boolean met = false;
 
         // 170.315 (h)(1)
         if (this.criteriaMet.containsKey("170.315 (h)(1)")) {
             this.counts.put("criteriaDpRequiredMet", 1);
+            met = true;
         }
 
         // 170.315 (h)(2)
         if (this.criteriaMet.containsKey("170.315 (h)(2)")) {
             this.counts.put("criteriaDpRequiredMet", 1);
+            met = true;
+        }
+        
+        if(!met){
```

Comment:
Checkstyle will complain about not having a space between the 1 and the (. Make sure you look at the checkstyle (and hopefully findbugs) errors.